### PR TITLE
APM-312140 - Added parser for AWS App Runner logs

### PR DESCRIPTION
**Pull request by bmrozinski-dt**
- application logs and service logs are covered
- also added attributes checks to unit test

### CODE REVIEW

**Comment by dyladan** [comment]
[CLA assistant check](https://cla.oss.dynalabs.io/dynatrace-oss/dynatrace-aws-log-forwarder?pullRequest=29) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.oss.dynalabs.io/dynatrace-oss/dynatrace-aws-log-forwarder?pullRequest=29) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.oss.dynalabs.io/check/dynatrace-oss/dynatrace-aws-log-forwarder?pullRequest=29) it.</sub>

File: tests/unit/full_flow/test_full_flow.py [hunk 1]
```diff
@@ -30,6 +32,26 @@
         "DEBUG": "false",
     })
 @pytest.mark.parametrize("testcase", [
+    ({
+        "lambda_event_file": THIS_DIR / "../../../examples/app_runner/kinesis_request_app_runner_application.json",
```
Comment (mmajcher) [inline tests/unit/full_flow/test_full_flow.py:36]: I think you went into wrong place with the test, please check out https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/blob/master/tests/unit/logs/test_transformation_full.py 
This has metadata attributes validation

New config I think does not need this ful_flow test

Comment (bmrozinski-dt) [reply tests/unit/full_flow/test_full_flow.py:36]: oh! damn 😂 Now I see... I missed tests in the `logs` subfolder
OK, I revert changes in the full flow test and extend the transformation test.

File: .gitallowed [hunk 2]
```diff
@@ -0,0 +1,2 @@
+(AIDA12345667789ABCEF)
```
Comment (mmajcher) [inline .gitallowed:1]: Why is this needed?
Otherwise all good

File: src/config/apprunner.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          "key": "aws.resource.id",
+          "pattern": "resource_id"
+        },
+        {
+          "key": "aws.arn",
+          "pattern": "format_arn('arn:{}:apprunner:{}:{}:service/{}/{}', [partition, region, account_id, resource_id, service_id])"
+        },
+        {
+          "key": "severity",
+          "pattern": "if( starts_with(log_content, 'ERROR'), &'ERROR', &if( starts_with(log_content, 'WARN'), &'WARN', &'INFO', @), @)"
```
Comment (dt-be) [inline src/config/apprunner.json:31]: Minor, I'd have a test case for it. Mostly as a doc of what we have handled. 
I wanted to see the logs in the jxw, but can't find them, I suppose you have them in some other env.
Major, thank you for all the examples;D

File: .gitallowed (hunk 2 above)
Comment (bmrozinski-dt) [reply .gitallowed:1]: This is the way we communicate to our git-secrets plugin that these credentials are fake (safe to reveal) and the plugin shouldn't be worried about these "secrets".

**Review by wojtasskorcz — DISMISSED** [review]

File: .gitallowed (hunk 2 above)
Comment (mmajcher) [reply .gitallowed:1]: But are those used anywhere in here? Can't find it

Comment (bmrozinski-dt) [reply .gitallowed:1]: In `test_transformation_full.py`

**Review by mmajcher — DISMISSED** [review]

**Review by dt-be — APPROVED** [review]

**Review by wojtasskorcz — APPROVED** [review]
